requests_auth/authentication.py

```diff
@@ -273,15 +299,15 @@ def __call__(self, r):
 
     def request_new_token(self) -> tuple:
         # As described in https://tools.ietf.org/html/rfc6749#section-4.4.3
-        token, expires_in = request_new_grant_with_post(
+        token, expires_in, _refresh_token = request_new_grant_with_post(
```

Comment:
If we don't use it later on:
```suggestion
        token, expires_in, _ = request_new_grant_with_post(
```